apis/router/src/prices.ts

```diff
@@ -241,7 +241,7 @@ export function comparePrices(
         totalR += r
         maxR = Math.max(r, maxR)
         levels.forEach((l, i) => {
-          if (r > l) levelsEx[i] += 1
+          if (r > l) levelsEx[i] = (levelsEx[i] || 0) + 1
```

Comment:
we know this is set to 0 already "const levelsEx = levels.map((_) => 0)"

Reply:
doesn't really matter as long as ts is happy